ceph.spec.in

```diff
@@ -33,10 +33,12 @@
 %bcond_with selinux
 %endif
 
-# LTTng-UST enabled on Fedora, RHEL 6+, and SLES 12+ and openSUSE TW
+# LTTng-UST enabled on Fedora, RHEL 6+, and SLES 12+ (not openSUSE)
 %if 0%{?fedora} || 0%{?rhel} >= 6 || 0%{?suse_version} >= 1315
+%if ! 0%{?is_openSUSE}
```

Comment:
There is a comment in (1) saying:
"Grouping with parentheses will fail on fedora/rhel/centos" 
This is the reason why I used successive %if in my patch.

(1) https://en.opensuse.org/openSUSE:Build_Service_cross_distribution_howto#Detect_a_distribution_flavor_for_special_code